In [ ]:
from google.colab import drive
drive.mount('/content/drive')

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


In [ ]:
# JME COUNTRY ESTIMATES MAY2022 (SURVEY ESTIMATES)
import pandas as pd
df = pd.read_csv('/content/drive/MyDrive/CSE303/12.csv', encoding = "Windows-1252")
df

ISO code Country and areas Survey year  Year* United Nations Region  \
0            AFG       AFGHANISTAN        1997   1997                  Asia   
1            AFG       AFGHANISTAN        2004   2004                  Asia   
2            AFG       AFGHANISTAN        2013   2013                  Asia   
3            AFG       AFGHANISTAN        2018   2018                  Asia   
4            ALB           ALBANIA     1996-98   1997                Europe   
...          ...               ...         ...    ...                   ...   
1030         ZWE          ZIMBABWE        2009   2009                Africa   
1031         ZWE          ZIMBABWE     2010-11   2010                Africa   
1032         ZWE          ZIMBABWE        2014   2014                Africa   
1033         ZWE          ZIMBABWE        2015   2015                Africa   
1034         ZWE          ZIMBABWE        2019   2019                Africa   

     United Nations Sub-Region                      SDG Region UNICEF Region  \
0                Southern Asia  Central Asia and Southern Asia            SA   
1                Southern Asia  Central Asia and Southern Asia            SA   
2                Southern Asia  Central Asia and Southern Asia            SA   
3                Southern Asia  Central Asia and Southern Asia            SA   
4              Southern Europe     Northern America and Europe           ECA   
...                        ...                             ...           ...   
1030            Eastern Africa              Sub-Saharan Africa           SSA   
1031            Eastern Africa              Sub-Saharan Africa           SSA   
1032            Eastern Africa              Sub-Saharan Africa           SSA   
1033            Eastern Africa              Sub-Saharan Africa           SSA   
1034            Eastern Africa              Sub-Saharan Africa           SSA   

     UNICEF Sub-Region WHO Region  ... Stunting Stunting Footnote  \
0                   SA       EMRO  ...     53.2             cw 10   
1                   SA       EMRO  ...     59.3              w 11   
2                   SA       EMRO  ...     40.4                 r   
3                   SA       EMRO  ...     38.2                 r   
4                 EECA       EURO  ...     20.4                 c   
...                ...        ...  ...      ...               ...   
1030               ESA       AFRO  ...     35.1               NaN   
1031               ESA       AFRO  ...     32.2               NaN   
1032               ESA       AFRO  ...     27.6               NaN   
1033               ESA       AFRO  ...     27.1               NaN   
1034               ESA       AFRO  ...     23.5               NaN   

     WAZ Survey Sample (N) Underweight Underweight Footnote  \
0                   4846.0        44.9                    c   
1                    946.0        32.9                 w 11   
2                4426469.0        24.6                    s   
3                  19539.2        19.1                    s   
4                   7642.0         7.1                    c   
...                    ...         ...                  ...   
1030                6195.5        11.7                    s   
1031                5413.6        10.2                  NaN   
1032                9591.3        11.2                    s   
1033                6380.0         8.5                  NaN   
1034                5977.8         9.7                  NaN   

                Fieldwork period  \
0          January,1997-May,1997   
1             May,2004-June,2004   
2          May,2013-October,2013   
3       March,2018-November,2018   
4     January,1996-December,1998   
...                          ...   
1030         April,2009-May,2009   
1031   September,2010-March,2011   
1032    February,2014-April,2014   
1033     July,2015-December,2015   
1034     January,2019-April,2019   

                                          Report Author  \
0                

In [ ]:
df.tail()

ISO code Country and areas Survey year  Year* United Nations Region  \
1030         ZWE          ZIMBABWE        2009   2009                Africa   
1031         ZWE          ZIMBABWE     2010-11   2010                Africa   
1032         ZWE          ZIMBABWE        2014   2014                Africa   
1033         ZWE          ZIMBABWE        2015   2015                Africa   
1034         ZWE          ZIMBABWE        2019   2019                Africa   

     United Nations Sub-Region          SDG Region UNICEF Region  \
1030            Eastern Africa  Sub-Saharan Africa           SSA   
1031            Eastern Africa  Sub-Saharan Africa           SSA   
1032            Eastern Africa  Sub-Saharan Africa           SSA   
1033            Eastern Africa  Sub-Saharan Africa           SSA   
1034            Eastern Africa  Sub-Saharan Africa           SSA   

     UNICEF Sub-Region WHO Region  ... Stunting Stunting Footnote  \
1030               ESA       AFRO  ...     35.1               NaN   
1031               ESA       AFRO  ...     32.2               NaN   
1032               ESA       AFRO  ...     27.6               NaN   
1033               ESA       AFRO  ...     27.1               NaN   
1034               ESA       AFRO  ...     23.5               NaN   

     WAZ Survey Sample (N) Underweight Underweight Footnote  \
1030                6195.5        11.7                    s   
1031                5413.6        10.2                  NaN   
1032                9591.3        11.2                    s   
1033                6380.0         8.5                  NaN   
1034                5977.8         9.7                  NaN   

               Fieldwork period  \
1030        April,2009-May,2009   
1031  September,2010-March,2011   
1032   February,2014-April,2014   
1033    July,2015-December,2015   
1034    January,2019-April,2019   

                                          Report Author  \
1030     Zimbabwe National Statistics Agency (ZIMSTAT).   
1031  Zimbabwe National Statistics Agency (ZIMSTAT) ...   
1032     Zimbabwe National Statistics Agency (ZIMSTAT).   
1033  Zimbabwe National Statistics Agency; ICF Inter...   
1034  Zimbabwe National Statistics Agency (ZIMSTAT) ...   

                                                 Source  Short Source  \
1030  Zimbabwe multiple indicator monitoring survey ...          MICS   
1031  Zimbabwe demographic and health survey 2010-11...           DHS   
1032  Zimbabwe Multiple Indicator Cluster Survey 201...          MICS   
1033  Zimbabwe Demographic and Health Survey 2015: F...           DHS   
1034  Zimbabwe Multiple Indicator  Cluster Survey 20...          MICS   

      U5 Population ('000s)  
1030            1938.279053  
1031            1997.281006  
1032            2236.182129  
1033            2258.788086  
1034            2138.020020  

[5 rows x 36 columns]

In [ ]:
print(df.info())

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1035 entries, 0 to 1034
Data columns (total 36 columns):
 #   Column                            Non-Null Count  Dtype  
---  ------                            --------------  -----  
 0   ISO code                       1035 non-null   object 
 1   Country and areas                 1035 non-null   object 
 2   Survey year                       1035 non-null   object 
 3   Year*                             1035 non-null   int64  
 4   United Nations Region             1035 non-null   object 
 5   United Nations Sub-Region         1035 non-null   object 
 6   SDG Region                        1035 non-null   object 
 7   UNICEF Region                     1023 non-null   object 
 8   UNICEF Sub-Region                 1023 non-null   object 
 9   WHO Region                        1027 non-null   object 
 10  World Bank Income Classification  1014 non-null   object 
 11  World Bank Region                 1035 non-null   object 
 12  LDC   

The above result shows, there are 1034 rows and 36 Columns. Ther are 

In [ ]:
#stunting_wasting_overweight
import pandas as pd
dff = pd.read_csv('/content/drive/MyDrive/CSE303/stun_was_over_database.csv')
dff.head()

Unnamed: 0    ISO Countries and areas UNICEF Regions Survey Years   Year*   \
0         NaN    NaN                 NaN            NaN          NaN      NaN   
1         3.0   AFG         Afghanistan             SA          2013   2013.0   
2      3076.0   AFG         Afghanistan             SA          2018   2018.0   
3         5.0   ALB             Albania           EECA          2000   2000.0   
4         6.0   ALB             Albania           EECA          2005   2005.0   

  Short Source                                        Full Source  \
0          NaN                                                NaN   
1          NNS         Afghanistan National Nutrition Survey 2013   
2        Other                     Afghanistan Health Survey 2018   
3         MICS  Multiple indicator cluster survey report Alban...   
4         MICS  Albania multiple indicator cluster survey 2005...   

      Latest Estimate Estimate Type            Wasted Only  \
0                 NaN           NaN  (WHZ <-2 but HAZ ≥-2)   
1   Historical Trend    Reanalyzed                     6.4   
2    Latest Estimate    Reanalyzed                     3.5   
3   Historical Trend    Reanalyzed                     9.4   
4   Historical Trend    Reanalyzed                     6.6   

      Wasted and Stunted                   Stunted Only  \
0  (WHZ <-2 AND HAZ <-2)  HAZ<-2 but WHZ≥-2 AND WHZ ≤+2   
1                    3.2                           33.8   
2                    1.6                           34.3   
3                    2.8                           19.8   
4                    0.7                           13.8   

  Stunted and Overweight       Overweight Only  \
0   WHZ > +2 but HAZ ≥-2  WHZ > +2 but HAZ ≥-2   
1                    3.4                   1.9   
2                    2.3                   1.8   
3                   16.6                  13.5   
4                   12.2                  12.6   

  Free from Wasting, Overweight, Stunting  
0          WHZ≥-2 AND WHZ ≤+2 AND HAZ ≥-2  
1                                    51.3  
2                                    56.5  
3                                    37.9  
4                                    54.1

In [ ]:
dff.tail()

Unnamed: 0    ISO Countries and areas UNICEF Regions Survey Years  \
627       789.0   ZWE            Zimbabwe            ESA          2009   
628       790.0   ZWE            Zimbabwe            ESA       2010-11   
629       791.0   ZWE            Zimbabwe            ESA          2014   
630       822.0   ZWE            Zimbabwe            ESA          2015   
631      2445.0   ZWE            Zimbabwe            ESA          2019   

      Year*  Short Source                                        Full Source  \
627   2009.0         MICS  Zimbabwe multiple indicator monitoring survey ...   
628   2010.0          DHS  Zimbabwe demographic and health survey 2010-11...   
629   2014.0         MICS  Zimbabwe Multiple Indicator Cluster Survey 201...   
630   2015.0          DHS  Zimbabwe Demographic and Health Survey 2015: F...   
631   2019.0         MICS  Zimbabwe Multiple Indicator  Cluster Survey 20...   

        Latest Estimate Estimate Type Wasted Only Wasted and Stunted  \
627   Historical Trend    Reanalyzed          1.5                0.9   
628   Historical Trend    Reanalyzed          2.3                0.8   
629   Historical Trend    Reanalyzed          1.9                1.3   
630   Historical Trend    Reanalyzed          2.6                0.7   
631    Latest Estimate    Reanalyzed          1.8                1.1   

    Stunted Only Stunted and Overweight Overweight Only  \
627         32.9                    1.2             2.3   
628         29.1                    2.2             3.5   
629         25.1                    1.2             2.4   
630         24.4                    2.0             3.6   
631         21.9                    0.5             2.0   

    Free from Wasting, Overweight, Stunting  
627                                    61.2  
628                                    61.9  
629                                    68.1  
630                                    66.7  
631                                    72.7

In [ ]:

dff.info()


<class 'pandas.core.frame.DataFrame'>
RangeIndex: 632 entries, 0 to 631
Data columns (total 16 columns):
 #   Column                                   Non-Null Count  Dtype  
---  ------                                   --------------  -----  
 0   Unnamed: 0                               631 non-null    float64
 1   ISO                                      631 non-null    object 
 2   Countries and areas                      631 non-null    object 
 3   UNICEF Regions                           631 non-null    object 
 4   Survey Years                             631 non-null    object 
 5    Year*                                   631 non-null    float64
 6   Short Source                             631 non-null    object 
 7   Full Source                              631 non-null    object 
 8   Latest Estimate                          631 non-null    object 
 9   Estimate Type                            631 non-null    object 
 10  Wasted Only                              632 non-n